src/components/ebay-pagination/index.js

```diff
@@ -91,46 +91,42 @@ function refresh() {
     for (let i = 0; i < this.state.items.length; i++) {
         if (this.state.items[i].current) {
             current = i;
-        } else {
-            // remove all hidden attribues to get accurate widths
-            this.pageEls[i].removeAttribute('hidden');
         }
+        this.pageEls[i].removeAttribute('hidden');
     }
+
     const totalPages = this.pageEls.length;
-    const pageNumWidth = this.pageEls[current].offsetWidth + constants.margin;
-    const containerWidth = this.containerEl.offsetWidth - pageNumWidth * 2;
-    const numPagesAllowed = Math.floor((containerWidth) / (pageNumWidth));
-    const adjustedNumPages = Math.min(constants.maxPagesAllowed - 1,
-        Math.max(numPagesAllowed, constants.minPagesRequired));
+    const pageNumWidth = 56;
```

Comment:
If we're hardcoding the width in CSS why do we have to do it here too?  Can't we reliably pick it up from the element now?

Reply:
hmm so far when measuring the first element they keep showing up 2 px wider on the 640px wide test.  Trying to track down why.  When I test this in a live view they never report anything other than 48, but in the test they keep showing up 50.  I wouldn't think this was possible with the updated css.